src/app/validators/credit-max.ts

```diff
@@ -0,0 +1,17 @@
+import { AbstractControl } from '@angular/forms';
+
+import { environment } from '../../environments/environment';
+
+export function ValidateCreditMax(control: AbstractControl) {
```

Comment:
As both validators are near identical to existing code, it would probably make sense to DRY this up. You could introduce a simple wrapper "thunk" that returns the fn, perhaps also explicitly setting up `ValidatorFn` return values as covered in part of https://blog.angular-university.io/angular-custom-validators/

Reply:
Good you reminded me I actually don't need the `ValidateCreditMin` validator because I directly use `Validators.min(0)` instead. So I'll delete `ValidateCreditMin` and keep `ValidateCreditMax` as is.

Reply:
Nice suggestion, done.